i18n.json

```diff
@@ -1103,5 +1103,15 @@
         "term": "WINDOW",
         "wAR": "نافذة",
         "wEN": "window"
+    },
+    {
+        "term": "AUTOONOMOUS_SYSTEM",
+        "wAR": "نظام مستقل",
+        "wEN": ["AS","Autonomous System"]
+    },
+    {
+        "term": "Address Allocation Protocol",
```

Comment:
```suggestion
        "term": "ADDRESS_ALLOCATION_PROTOCOL",
```

Reply:
The "term" value of  uses this format to fetch data (this part is hidden)